ms2query/networking.py

```diff
@@ -70,3 +76,136 @@ def do_networking(query_id, matches, similarity_matrix):
     lib_ids = matches.index.tolist()
     network = add_library_connections(init_network, similarity_matrix, lib_ids)
     return network
+
+
+def draw_edges(network, pos, attribute, attr_cutoff, width_default,
+               style="solid", cmap=None, edge_labels=True):
+    """
+    Draw edges where width is determined by attribute score, if score > cutoff
+
+    Args:
+    -------
+    network: nx.Graph
+        Network for which to draw edges
+    pos: dict of {node: [x,y]}
+        Contains coordinates for each node
+    attribute: str
+        The attribute that should be looked at for the score
+    attr_cutoff: float
+        Cutoff for the attribute scores
+    width_default: int
+        The max width of an edge
+    style: str, optional
+        The style of the edges. Default = solid
+    cmap: matplotlib.colourmap, optional
+        If provided, a cmap to colour the edges by attribute score.
+        Default = none
+    edge_labels: bool, optional
+        Plot edge labels
+    """
+    labels = nx.get_edge_attributes(network, attribute)
+    max_val = max(labels.values())
+    if max_val < 1:
+        max_val = 1  # to keep 1 always the max value for scores: s2v, cosine..
```

Comment:
Are there instances that ``max(labels.values())`` becomes > 1 ?
Should not be possible, right?
Because than it could immediately be set to 1 always.

Reply:
For now yes, as the cosine matches are ints higher than 1 for example. You can see it in the plot

Reply:
Oh yes, just thought about the actual scores. Fine then.